inc/osvr/Server/ConfigureServerFromFile.h

```diff
@@ -38,130 +38,30 @@
 #include <exception>
 #include <fstream>
 #include <iostream>
-#include <sstream>
+#include <vector>
 
```

Comment:
Need `#include <string>`. And it looks like you can ditch `iostream` and `fstream` and `exception` (move 'em to the `.cpp` file). Same with the Log headers above.